src/ngraph_confirm_pass.cc

```diff
@@ -21,6 +21,20 @@
 
 using namespace std;
 namespace ngraph_bridge {
+namespace detail {
+// yes, these have to be here
+// defined in a function or even in a class the symbols cannot be found for
+// template resolution
+static constexpr char FILL_DIMS[] = "_ngraph_fill_static_dims";
```

Comment:
While we're here, can we go ahead and tweak `ngraph_builder.cc` to use these constants as well?

Reply:
That's a great idea. Why didn't I think of that!